### PR TITLE
Fix redeem issue

### PR DESCRIPTION
**Pull request by alannotnerd**

### CODE REVIEW

File: pallets/loans/src/lib.rs [hunk 1]
```diff
@@ -1429,7 +1431,7 @@ impl<T: Config> Pallet<T> {
         })?;
 
         T::Assets::transfer(asset_id, &Self::account_id(), who, redeem_amount, false)
-            .map_err(|_| Error::<T>::InsufficientMarketLiquidity)?;
+            .map_err(|_| Error::<T>::InsufficientCash)?;
```
Comment (GopherJ) [inline pallets/loans/src/lib.rs:1434]: do we still need `InsufficientMarketLiquidity`

File: pallets/loans/src/tests.rs [hunk 2]
```diff
@@ -274,7 +271,7 @@ fn redeem_fails() {
 }
 
 #[test]
-fn withdraw_fails_when_insufficient_liquidity() {
+fn redeem_fails_when_insufficient_liquidity() {
```
Comment (GopherJ) [inline pallets/loans/src/tests.rs:274]: should be in another unit tests, this is for testing normal redeem

Comment (alannotnerd) [reply pallets/loans/src/tests.rs:274]: done

File: pallets/loans/src/lib.rs (hunk 1 above)
Comment (alannotnerd) [reply pallets/loans/src/lib.rs:1434]: I think `InsufficientMarketLiquidity` should only appear in `ensure_liquidity`. Alternatively, We can add a new error for `PalletAccountBalanceLow` for this situation.

**Pull request merged**